include/ttmlir/OpModel/TTNN/TTNNOpModel.hpp

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
```

Comment:
nit: Currently all our header files end with `.h` and we use `ifndef...` header guards instead of pragma once. I don't have a strong opinion on which approach we should use, but it would be nice to keep it consistent.